enterprise/app/trends/trends.css

```diff
@@ -59,3 +59,25 @@
 .heatmap-svg {
   user-select: none;
 }
+
+.drilldown-chart {
+  display: inline-block;
+}
+
+.drilldown-chart-title {
+  margin-left: 4px;
+  font-weight: 600;
+}
+
+.dd-tt-base {
+  color: #8884d8;
+}
+
+.dd-tt-selected {
+  color: #82ca9d;
+}
+
+.select.drilldown-select {
+  font-size: 16px;
+  margin: 0 16px;
+}
```

Comment:
nit: can you prefix all these with `.drilldown-page ` (since we don't have local CSS yet :sweat_smile: ) also the heatmap rules above, maybe

A lot of our existing CSS is pretty inconsistent about this, but I'd like to have some sort of minimal prefixing convention at least going forward, until we have local CSS

Reply:
`heatmap-svg` above is the only heatmap-specific css class, which seems fine to me for now (let me know if there's a different prefix you want there, though).  updated the rest.